src/pages/guides/uxp_guide/uxp-misc/manifest-v5/index.md

```diff
@@ -0,0 +1,282 @@
+  
+```json
+{
+    "requiredPermissions": {
+        "localFileSystem": "request"
+    }
+}
+```
+ 
+```js
+const fs = require('uxp').storage.localFileSystem;
+const file = await fs.getFileForSaving('manifest_demo.txt');
+await file.write('Manifest v5 demo');
+```
+If `plugin` is not specified for plugin storage, the manifest will default to that setting.
+
+### Launch Process
+The `launchProcess` permission in the manifest controls the ability to launch applications and open files in other applications. 
+
+Before manifest v5, `shell.openPath()` was enabled using the feature flag `kEnableShellOpenPath`. Additionally, `openExternal()` was only accessible with the following code block in manifest: 
+```json
+"permissions": {        
+    "launchProcess": 
+        "request" | "any"    
+}
+```
+
+With manifest v5, the feature flag has been removed by v5 and the `launchProcess` permission must be declared to use `openExternal()` or `openPath()`. 
+```json
+"permissions": {        
+    "launchProcess": {    
+        // allows launching files with specified URI schemes         
+        "schemes": 
+            [ "https", "slack", "adbxd" ],   
+        // allows opening files with the specified file extensions          
+        "extensions": 
+            [ ".pdf", ".xd", ".ps" ],        
+    },   
+}
+```
+Both openPath() and openExternal() rely on this permission set, and upon either function call, the user will get a runtime consent dialog. Only after they agree will the API call execute. 
+
+### Plugin communication 
+If enabled, the plugin can communicate with other installed plugins. Defaults to `false`.
+```json
+"permissions": {        
+    "ipc": {
+        "enablePluginCommunication": true   
+    }
+}
+```
+```js
+const { pluginManager } = require("uxp");
+// find the Alchemist plugin in the loaded plugins
+const alchemistPlugin = Array.from(pluginManager.plugins).find(plugin => plugin.id==="2bcdb900");   
+//What commands and panels are available?
+const alchemistCommands = Array.from(alchemistPlugin.manifest.commands, command => command.commandId); // result: ["resetStateFn"]
+const alchemistPanels = Array.from(alchemistPlugin.manifest.panels, command => command.panelId); // result: ["inspector"]     
+// Show the inspector panel; note that panels can only be made visible -- you can't ask to hide the panel
+alchemistPlugin.showPanel("inspector");
+// Reset Alchemists state... but be sure you want to do this!
+alchemistPlugin.invokeCommand("resetStateFn");
+```
+
+### Host Communication 
+If enabled, the plugin can communicate with installed filters, automation plugins, etc. Defaults to `false`.
+```json
+"permissions": {        
+    "ipc": {
+        "enableHostCommunication": true   
+    }
+}
+```
+
+### Fonts 
+This permission allows the plugin to access the fonts installed on the system. 
+```json
+"permissions": {        
+    "fonts": "readInstalled" 
+}
+```
+
```

Comment:
For 242-249 -- double check w/ Eng, but I don't think we actually have an API yet for this access. IIRC, the permision here is a little in advance of the of the API access itself.

Reply:
I'll check with Bharat